standards/certification/getting-scs-compatible-certified.md

```diff
@@ -0,0 +1,114 @@
+
+## Declarations
+
+For the SCS-compatible IaaS v5 standard, the providers must -- if they implement availability zones
+at all (which is optional) -- guarantee certain levels of independence for these. This can not
+be fully tested by an automated test. The process thus envisions that providers must create some
+documentation on the physical infrastructure and how it maps to availability zones and declare that
+this documentation reflects the truth. SCS will review the docs and judge whether they meet the
+criteria. In case of doubt, audits can be performed.
+
+## Forum SCS-Standards @ OSBA
+
+The SCS brand belongs to the Open Source Business Alliance e.V. (OSBA), an non-profit organization and
+association for the Open Source Industry in Germany. The OSBA creates the Forum SCS-Standards
+which performs the work to evolve the SCS standards, develops the tests and perform the certification
+process.
+
+Members of the OSBA can become also member of the Forum SCS-Standards for an additional membership
+fee, providing the financial resources for the Forum to do its work. Membership in the OSBA is
+open to any organization that supports the goals of the OSBA.
+Alternatively, a certification fee can be paid without any membership; the fee corresponds to the
+lower tier membership fee.
+
+## Getting listed and tested
+
+When all tests are passing, all needed declarations are done, fees for the certification or the
+membership in the Forum SCS-Standards at the OSBA have been paid, the infrastructure service
+can become officially certified.
+
+The SCS team will add the cloud to the [list of certified clouds](https://docs.scs.community/standards/certification/overview)
+on the SCS docs page. This can be used to prove to customers that the cloud is SCS compliant.
+Note that there will be a nightly job that tests the cloud for compliance, which will be
+triggered by SCS infrastructure (zuul). For this, access to a tenant on the cloud needs
+to be provided free of charge. (This only requires very low quota, one VM is created for a minute
+in one of the tests.)
+The list of certified clouds will be replaced by the
+[compliance monitor](https://compliance.sovereignit.cloud/page/table) soon.
+
+For clouds not being accessible from the outside, a VPN tunnel or a local monitoring
+job (with result upload) can be used.
+
+Next to the addition into the list, we also plan to create an SCS-certified badge that
+operators will be allowed to use in their marketing material as long as they fulfill the
+certification conditions.
+
+Note that for almost all certified clouds in the list of certified clouds, we also
+have a health monitor running (currently still
+[openstack-health-monitor](https://docs.scs.community/docs/operating-scs/guides/openstack-health-monitor/Debian12-Install)
+but soon the new [health-monitor](https://scs.community/tech/2024/09/06/vp12-scs-health-monitor-tech-preview/)),
```

Comment:
I think it is not good practice to talk about future plans in such a document. Document the "as is" case and update if the "as is" case changes.